### ISSUE TITLE
Asking : Does this BOT could be added to group chat or such?

### ISSUE THREAD

**Issue by benyaminl** [avcwisesa/gitlab-utility-linebot#6]
Hello, I want to ask, does this bot for line.me? If yes, then will it work for like notifiying issue, merge request, and etc for line? Thanks

**Comment by avcwisesa** [comment]
Hello @benyaminl! Yes, this bot is indeed for line.me, and can be added to group chats. It's actually intended to do so in the first place :)

Right now notifying for events is not yet implemented in this bot, but it can refer to MR and issue you specify. For example if you typed !4914 it will give you the link to MR with that number in your project

**Comment by benyaminl** [comment]
@avcwisesa is there any public bot available? I mean to add to group chat?

**Comment by avcwisesa** [comment]
No, unfortunately because you have to specify the project on config, you have to host the bot yourself
